src/Adapter/Build/Common/MSTest.TestAdapter.props

```diff
@@ -2,8 +2,7 @@
 
   <PropertyGroup>
     <EnableMSTestV2CopyResources Condition=" '$(EnableMSTestV2CopyResources)' == '' ">true</EnableMSTestV2CopyResources>
-    <EnableMSTestRunner Condition=" '$(EnableMSTestRunner)' == '' ">false</EnableMSTestRunner>
-    <IsTestingPlatformApplication>$(EnableMSTestRunner)</IsTestingPlatformApplication>
```

Comment:
@MarcoRossignoli I don't see this property in the SDK configuration. I think this is a bug but I would like confirmation.

Reply:
Not needed because it will be set by this props when the adpater will be imported by the MSTest sdk.